### ISSUE TITLE
Development Install Issues

### ISSUE THREAD

**Issue by droctothorpe** [dask/dask-labextension#117]
I'm running through the [development install](https://github.com/dask/dask-labextension#development-install) steps in the README.

1) It assumes typescript is installed. It should include `jlpm add typescript` (unless you prefer a different method for installing typescript).

2) `jlpm run build` fails with the following error:
```
❯ jlpm run build   
yarn run v1.15.2
$ tsc
src/index.ts:301:45 - error TS2533: Object is possibly 'null' or 'undefined'.

301       sidebar.dashboardLauncher.input.url = settings.get('defaultURL')
                                                ~~~~~~~~

src/index.ts:307:25 - error TS2533: Object is possibly 'null' or 'undefined'.

307       autoStartClient = settings.get('autoStartClient').composite as boolean;
                            ~~~~~~~~

src/index.ts:312:5 - error TS2533: Object is possibly 'null' or 'undefined'.

312     settings.changed.connect(onSettingsChanged);
        ~~~~~~~~


Found 3 errors.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Digging through docs (haven't written TypeScript before), but want to call out this immediate road block for potential contributors. There's always the possibility that I'm installing or configuring something incorrectly. That's never happened before 😉 .

**Comment by droctothorpe** [comment]
This can be closed. I reinstalled and it worked fine.

**Comment by ian-r-rose** [comment]
Thanks for following up @droctothorpe, I suspect this was related to the JupyterLab 2.0 rollout, and should be fixed by #119.
